README.md

```diff
@@ -301,6 +301,55 @@ See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Automati
 </details>
 
 
+## Debug
```

Comment:
Please, drop this from main README.md

It's not important information for end users. Main README should highlight product capabilities, but not developer tips and tricks.

Reply:
Where can I write this information?

Reply:
somewhere in src/docs you can create page DEBUGGING.md

Reply:
Moved to  tools/llm_bench/README.md